build-system/babel-plugins/babel-plugin-transform-inline-configure-component/index.js

```diff
@@ -221,3 +224,9 @@ module.exports = function ({types: t}) {
     },
   };
 };
+
+// Even though we are using the path module, JS Modules should never have
+// their paths specified in Windows format.
+function toPosix(filepath) {
+  return filepath.replace(/\\\\/g, '/').replace(/\\/g, '/');
+}
```

Comment:
Instead of making multiple copies of this function, would it be possible to use one of the functions under `path.win32` as described in https://nodejs.org/docs/latest-v12.x/api/path.html#path_windows_vs_posix?

If that doesn't work, perhaps use a package like https://www.npmjs.com/package/slash?

Reply:
Ah. Actually I think we can replace all of the `require('path')` expressions with `require('path').posix`! Nice find

Reply:
I was able to remove most of the `toPosix` functions, but was left with two that I couldn't.
I'd rather not pull in a lib, but would be fine with extracting to a shared fn if you think preferable.

Reply:
SGTM. Agree with not pulling in a lib, especially since this file affects the correctness of compiled code.